packages/hardhat-verify/src/internal/chain-config.ts

```diff
@@ -186,6 +186,14 @@ export const builtinChains: ChainConfig[] = [
       browserURL: "https://goerli.basescan.org/",
     },
   },
+  {
```

Comment:
The `builtinChains` array is sorted by `chainId`. This object needs to be placed after `ftmTestnet`, or the tests will fail.

Reply:
updated